core/robotstxt/module.go

```diff
@@ -1,36 +1,78 @@
 package robotstxt
 
 import (
-	"net/http"
-
 	"flamingo.me/dingo"
+	"flamingo.me/flamingo/v3/core/robotstxt/interfaces"
+	"flamingo.me/flamingo/v3/framework/web"
 )
 
 type (
 	// Module for robotstxt
-	Module struct {
-		DefaultMux *http.ServeMux `inject:",optional"`
-		Filepath   string         `inject:"config:core.robotstxt.filepath"`
+	Module struct{}
+
+	routes struct {
+		securityTxtActivated bool
+		humansTxtActivated   bool
+		files                interfaces.FileControllerInterface
 	}
 )
 
 // Configure DI
 func (m *Module) Configure(injector *dingo.Injector) {
-	if m.DefaultMux != nil {
```

Comment:
I think, for backwards compatibility, we still need the mechanism to support the DefaultMux here